modules/microshift-adding-repos-to-image-builder.adoc

```diff
@@ -11,15 +11,69 @@ Use the following procedure to add the {product-title} repositories to Image Bui
 .Prerequisites
 * Your build host meets the Image Builder system requirements.
 * You have installed and set up Image Builder and the `composer-cli` tool.
-* You have root user access to your build host.
+* You have root-user access to your build host.
 
 .Procedure
 
-* Enable the {product-title} RPM repositories on the build host by running the following command:
+. Enable the {product-title} RPM repositories on the build host by running the following command:
 +
-[source,terminal,subs="attributes+"]
+[source,terminal]
 ----
-$ sudo subscription-manager repos \
-    --enable rhocp-{ocp-version}-for-{rhel-major}-$(uname -i)-rpms \
-    --enable fast-datapath-for-{rhel-major}-$(uname -i)-rpms
+$ sudo cat > rhocp-4.13.toml <<EOF
```

Comment:
What directory should that file end up in?

I suspect using `sudo cat` is not going to do the right thing because of the way the shell interprets the redirection. We've used `sudo tee` in other places to get around that. On the other hand, it doesn't look like `sudo composer-cli sources add` below would require the file in this step be created by root, so maybe we don't need the sudo here at all?

Reply:
It was `# cat`, but the rest of the commands are `$ sudo`, so I wanted them to be consistent. This will come up in peer review, so let's figure out which way it needs to be. We have root access as a prereq for the entire procedure, so whichever command syntax is correct is fine.

Reply:
@ggiguash Do we have a directory to list for the end user? Or should that be specified somewhere in this command?

Reply:
I was able to run the `cat` step as a regular user and then use the file as root. It doesn't really matter where the file is, the command that loads it can read the file and it passes data to the service so the service does not need the file itself.

Reply:
@dhellmann So we don't need to answer your original question: "What directory should that file end up in?"

Reply:
Yes, it's not necessary to specify the file location. They are created in the current directory.